internal/appsec/listener/util/util_test.go

```diff
@@ -3,18 +3,14 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2016 Datadog, Inc.
 
-//go:build appsec
-// +build appsec
-
-package appsec
+package util
```

Comment:
Shall we choose a definite name between `util` and `shared`? In the case of this package I would say this is more a shared component than a utility component since `runWaf` is in this package

Reply:
Yeah I can go with `shared` for sure!

Reply:
+1 for `shared`